megamock/megamocks.py

```diff
@@ -273,6 +272,8 @@ def _generate_mock_name(
                         mock_name += "."
                     mock_name += spec_name
                 else:
+                    if mock_name:
+                        mock_name += " -> "
```

Comment:
The return value is represented as -> now. You may see something like this: ` -> str`